src/classes/CRLP_Query_SEL.cls

```diff
@@ -220,7 +220,16 @@
      * @return part of a SOQL string
      */
     private static String buildChildQuery(SObjectType childObject, String relationshipName) {
-        String childQuery = ', (' + buildObjectQueryForRollup(childObject).split(' FROM ')[0] + + ' FROM ' + relationshipName + ')';
+        String childQuery = ', (' + buildObjectQueryForRollup(childObject).split(' FROM ')[0] + + ' FROM ' + relationshipName;
```

Comment:
What is the double plus doing here? ``` + +```

Reply:
Good catch @lmeerkatz. Totally missed the two pluses.